src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -470,7 +470,7 @@ public function getEvent()
     public function hasPermission()
     {
         if (!$this->parameters->has('permission')) {
-            return true;
+            return false;
         }
 
         return false !== $this->parameters->get('permission');
```

Comment:
The method `hasPermission` is now used in a wrong context. See [RbacAuthorizationChecker](https://github.com/Sylius/Sylius/blob/147d0169768a91c08979cc6841cf5c1b7e9b5746/src/Sylius/Bundle/RbacBundle/Authorization/RbacAuthorizationChecker.php#L44). Taking into account current logic:

| $this->parameters->has('permission') | $this->parameters->get('permission')  | hasPermission |
| ------------- |------------- |------------
| false      | - | false |
| true      | false | false |
| true      | 'permission' | true |

When I do not set permission